pkg/packagekit/autoupdates.jsx

```diff
@@ -275,6 +288,17 @@ export default class AutoUpdates extends React.Component {
         if (!backend)
             return null;
 
+        if (!backend.installed) {
+            return (
+                <div className="header-buttons pk-updates--header pk-updates--header--auto" id="automatic">
+                    <h2 className="pk-updates--header--heading">{_("Automatic Updates")}</h2>
+                    <div className="pk-updates--header--actions">
+                        <button className="btn btn-default"
+                                onClick={() => { install_dialog(backend.packageName).then(() => this.initializeBackend(true), () => null) }}>{_("Enable")}</button>
```

Comment:
Do we really need a new UI for this? I thought clicking the On/Off switch could bring up the install dialog. It seems a bit confusing to click on "Enable" and then auto-updates are still "Off".

Reply:
That makes a lot more sense, will do it that way.